stablehlo/reference/ProcessGrid.h

```diff
@@ -42,15 +67,20 @@ namespace detail {
 /// Once all processes have added their data, the data in `values` is moved to
 /// `result` that multiple processes can concurrently read from.
 struct RendezvousState {
+ public:
```

Comment:
`public` is implied by `struct`

Reply:
Thanks, I missed updating this while debugging :)